00-programming-fundamentals/exercises/regex-exercises.md

```diff
@@ -0,0 +1,33 @@
+
+# Regex Practice
+
+## Matching Patterns
+
+Write a regular expression which would match the patterns on the left, but not the right.
+
+You can use [Regexpal](http://www.regexpal.com/) to craft Regular a Expression and compare it to sample text.
```

Comment:
I think we should use rubular b/c regexpal is for JS I believe

Reply:
The syntax is the same, and I haven't found a difference between them in the regex.